### ISSUE TITLE
stdout can not be sent into multiple channels

### ISSUE THREAD

**Issue by Hammarn** [nextflow-io/nextflow#388]
I can declare two channels as such:
```
output:
file *.txt into out_1, out_2
```

but not for stdout:
```
output:
stdout into stringtie_log, stringtie_stdout
```

It generates the following error:
```
ERROR ~ expecting '}', found ',' @ line 889, column 31.
       stdout into stringtie_log , stringtie_stdout
                                 ^
1 error
```
(edited)

**Comment by pditommaso** [comment]
This should related to #141

**Comment by Hammarn** [comment]
Using 
```
 stdout into (stringtie_log , stringtie_stdout)
```
results in another error: 

```
ERROR ~ No such variable: stringtie_log
```

**Comment by pditommaso** [comment]
The following works: 

```
output:
stdout() into (stringtie_log, stringtie_stdout)
```

But I agree there's something wrong here. I will investigate.

**Comment by Hammarn** [comment]
Right I missed the initial parenthesis.
This does _seem_ to work but it's somehow causing other odd issues. Suddenly I'm getting these error when running the pipeline:
``` 
ERROR ~ Channel `bam_md` has been used twice as an input by process `dupradar` and another operator
```
Which is not true, and it switches around which channel it is complaining about when I change other lines of the code and re-run it.

**Comment by pditommaso** [comment]
OH. If so I would suggest to redirect the stdout to a file and capture it in the output as workaround.

**Comment by ewels** [comment]
Can we just specify `.command.log` as an output? Or will that break things?

Side note - would be great to have `stderr` as an output type too. And `stdlog`? Workarounds for all of these things are easy enough, but it feels good to keep the script blocks clean 🤓

**Comment by pditommaso** [comment]
Yes, of course you can use `.command.log` that's just a like the other (at least I hope  😄). 

> Side note - would be great to have stderr as an output type too. And stdlog?

I think it could be done.

**Comment by pditommaso** [comment]
I've uploaded version snapshot that should fix this issue. You may want to give a try: 

```
$ NXF_VER=0.25.2-SNAPSHOT nextflow info 

  Version: 0.25.2-SNAPSHOT build 4463
  Modified: 29-06-2017 18:20 UTC (20:20 CEST)
  System: Linux 4.4.0-38-generic
```

**Comment by pditommaso** [comment]
Fixed in version `0.25.2`
